content/id/docs/concepts/scheduling/scheduling-framework.md

```diff
@@ -0,0 +1,251 @@
+tersebut.
+
+[kep]: https://github.com/kubernetes/enhancements/blob/master/keps/sig-scheduling/20180409-scheduling-framework.md
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+# Alur kerja kerangka kerja
+
+Kerangka kerja penjadwalan mendefinisikan beberapa titik ekstensi. _Plugin_ penjadwal
+mendaftar untuk dipanggil di satu atau lebih titik ekstensi. Beberapa _plugin_ ini
+dapat mengubah keputusan penjadwalan dan beberapa hanya bersifat informasi.
+
+Setiap upaya untuk menjadwalkan satu Pod dibagi menjadi dua fase, **
+Siklus Penjadwalan (_Scheduling Cycle_) ** dan ** Siklus Pengikatan (_Binding Cycle_) **.
+
+## Siklus Penjadwalan dan Siklus Pengikatan
+
+Siklus penjadwalan memilih sebuah Node untuk Pod, dan siklus pengikatan menerapkan
+keputusan tersebut ke klaster. Secara bersama-sama, siklus penjadwalan dan siklus 
+pengikatan diartikan sebagai sebuah "konteks penjadwalan (_scheduling context_)".
+
+Siklus penjadwalan dijalankan secara serial, sementara siklus pengikatan dapat 
+berjalan secara bersamaan.
+
+Siklus penjadwalan atau pengikatan dapat dibatalkan jika Pod telah ditentukan 
+untuk tidak terjadwalkan atau jika adanya kesalahan internal. Pod akan 
+dikembalikan ke antrian dan mencoba lagi.
+
+## Titik-titik ekstensi
+
+Gambar berikut menunjukkan konteks penjadwalan Pod dan titik-titik ekstensi
+yang diperlihatkan oleh kerangka penjadwalan. Dalam gambar ini "Filter" adalah
+setara dengan "Predicate" dan "Scoring" setara dengan "Priority Function".
+
+Satu _plugin_ dapat mendaftar di beberapa titik ekstensi untuk melakukan pekerjaan
+yang lebih kompleks atau _stateful_.
+
+{{< figure src="/images/docs/scheduling-framework-extensions.png" title="Titik-titik ekstensi dari kerangka kerja Penjadwalan" >}}
+
+### QueueSort {#queue-sort}
+
+_Plugin_ ini digunakan untuk mengurutkan Pod-Pod dalam antrian penjadwalan. _Plugin_
+QueueSort pada dasarnya menyediakan fungsi `Less (Pod1, Pod2)`. Hanya satu jenis
+_plugin_ QueueSort yang dapat diaktifkan dalam waktu yang sama.
+
+### PreFilter {#pre-filter}
+
+_Plugin_ ini digunakan untuk melakukan pra-proses informasi tentang Pod, atau untuk 
+memeriksa tertentu kondisi yang harus dipenuhi oleh klaster atau Pod. Jika 
+_plugin_ PreFilter menghasilkan hasil yang salah, siklus penjadwalan dibatalkan.
+
+### Filter
+
+_Plugin_ ini digunakan untuk menyaring Node yang tidak dapat menjalankan Pod. 
+Untuk setiap Node, penjadwal akan memanggil _plugin_ Filter sesuai dengan urutan
+mereka dikonfigurasi. Jika ada _plugin_ Filter menandai Node menjadi _infeasible_,
+maka _plugin_ yang lainnya tidak akan dipanggil Node itu. Node dapat dievaluasi 
+secara bersamaan.
+
+### PreScore {#pre-score}
+
+_Plugin_ ini digunakan untuk melakukan pekerjaan "pra-penilaian", yang menghasilkan 
+status yang dapat dibagi untuk _plugin_ Score untuk digunakan. Jika _plugin_ 
+PreScore mengeluarkan hasil salah, maka siklus penjadwalan dibatalkan.
+
+### Score {#scoring}
```

Comment:
This can actually be:
```suggestion
### Score
```
because the original fragment identifier mismatch is to handle existing incoming links, and for bahasa Indonesia there are no existing incoming links.